### ISSUE TITLE
Narrator + Edge doesn't read the title of the Dialog

### ISSUE THREAD

**Issue by YusongLiu** [OfficeDev/office-ui-fabric-react#2419]
### Bug Report

- __Package version(s)__: 4.25.1
- __Browser and OS versions__: Edge latest version

#### Describe the issue:

I use the `Dialog` component. I set the `title` of `dialogContentProps` of the `Dialog`. When I open the dialog, Narrator + Edge doesn't read the title.

#### Actual behavior:

Narrator + Edge doesn't read the title of the dialog when opening the Dialog.

#### Expected behavior:

Narrator + Edge should read the title of the dialog when opening the Dialog.

**Comment by mdahamiwal** [comment]
The fix went in with https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v4.30.1, try it out :)
